.circleci/config.yml

```diff
@@ -19,16 +19,28 @@ references:
 jobs:
   test:
     docker:
-      - image: swift:4.2
+       - image: vapor/swift:5.1
     steps:
       - checkout
       - run:
           name: Compile code
-          command: swift build
+          command: swift build --jobs 1
```

Comment:
Note to self: we might want the same for Stevenson, to avoid random failures we sometimes have during build of Vapor (which a re-run usually fixes)